libs/geography/src/selectors/country-entities.selector.ts

```diff
@@ -67,7 +76,11 @@ const createCountryEntitySelectors = <T extends DaffCountry>() => {
     /**
      * Selector for a specific country's subdivisions.
      */
-    selectCountrySubdivisions
+    selectCountrySubdivisions,
+    /**
+     * Selector for checking if a country has been fully loaded.
```

Comment:
I think you should mention subdivisions in this comment, since loading the subdivisions is what it means to be "fully loaded"

Reply:
added